lib/KBoxHardware/src/KBoxConfig.cpp

```diff
@@ -0,0 +1,52 @@
+/*
```

Comment:
I don't think we should put this in KBoxHardware because this folder is only for hardware support. The config is very specific to what kbox does so I would just put this into `src/common/config` (so it can be shared with the wifi module too).

Reply:
ok, done.
Just to keep in mind, that (as far as I know) if the config is in the src part then libraries can not access it, which means that we will *have to* do the axis- and sign thing in IMUService and not in the driver.
(I think for now it is perfect, and it could easily be changed later, if needed)